MANIFEST.in

```
@@ -0,0 +1 @@
+include LICENSE
```

It seems we have two license files in the source. Which one do we want to publish?

---

`LICENSE` was first commited. `LICENSE.md` has a different `Copyright` line which is a bit off in regard of the year.

---

We want to keep the markdown file.
```suggestion
include LICENSE.md
```